### PR TITLE
Allow capturing USIG identity from any UI

### PR DESCRIPTION
**Pull request by sergefdrv**
This pull request adjusts the way USIG identity is captured.

Related to #179.

### CODE REVIEW

File: sample/authentication/crypto.go [hunk 1]
```diff
@@ -217,7 +217,7 @@ func (au *SGXUSIGAuthenticationScheme) VerifyAuthenticationTag(m []byte, sig []b
 	// UIs. This, for example, can be achieved using some
 	// bootstrapping procedure.
 	epoch, ok := au.epoch[fingerprint]
-	if !ok && ui.Counter == uint64(1) {
+	if !ok {
```
Comment (nhoriguchi) [inline sample/authentication/crypto.go:219]: The above inline comment seems to mention the assumption about `ui.Counter == uint64(1)`. So does it need some update?

Comment (sergefdrv) [reply sample/authentication/crypto.go:219]: Yes, it seems so. I think we should distinguish here "first received UI" from "first generated UI". Is it then as simple as changing "generates its first UI" to "generates its UIs"?

**Comment by sergefdrv** [comment]
Summary of [changes](https://github.com/hyperledger-labs/minbft/compare/9640f09cc7b558f3d580295aacf44cff0e37557a..4208405688182614adc0f699eca10b57c7da92e8): updated comment.

File: sample/authentication/crypto.go (hunk 1 above)
Comment (nhoriguchi) [reply sample/authentication/crypto.go:219]: Thank you for the update.

**Review by nhoriguchi — APPROVED** [review]

**Pull request merged**